corehq/apps/app_manager/views/utils.py

```diff
@@ -639,7 +639,7 @@ def set_shadow_module_and_form_session_endpoint(
     shadow_module.session_endpoint_id = raw_endpoint_id
     shadow_module.form_session_endpoints = [
         ShadowFormMapping(form_id=m['form_id'], session_endpoint_id=m['session_endpoint_id'])
-        for m in form_session_endpoints
+        for m in form_session_endpoints if m['session_endpoint_id'] and m['session_endpoint_id'] != ""
```

Comment:
nit: `""` is falsy, so the second half of this conditional is redundant

Reply:
removed